libs/api/domains/work-machines/src/lib/models/machineDetails.ts

```diff
@@ -31,4 +31,7 @@ export class MachineDetails {
 
   @Field(() => String, { nullable: true })
   supervisorName?: string | null
+
+  @Field(() => Boolean, { nullable: true })
+  paymentRequiredForOwnerChange?: boolean | null
```

Comment:
```suggestion
  @Field({ nullable: true })
  paymentRequiredForOwnerChange?: boolean | null
```

Unnecessary for boolean
![image](https://github.com/island-is/island.is/assets/9449413/6d49730c-2d64-4d16-9e80-39c9b3b6c5c2)